rental.rb

```diff
@@ -0,0 +1,8 @@
+class Rental
+  attr_accessor :date, :person
+
+  def initialize(date, person)
+    @date = date
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] A rental belongs to a book and also belongs to a person, so please make sure to add the assocciations for book and rental here as well:

```ruby

class Rental
  # You can advise students to [OPTIONALLY] use shortcuts by giving them a link to the lesson they took: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor
  # Getters and Setters for @book and @person instance variables should be present
  attr_accessor :date, :book, :person
  .
  
  # *The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. Link to the example article: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  def initialize(date, book, person)
    .
    .

    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
  end
end


```